starknet_py/tests/e2e/utils.py

```diff
@@ -27,5 +33,16 @@ async def make_devnet_client(self) -> Client:
         client = await AccountClient.create_account(net=self.net, chain=self.chain)
         return client
 
+    def make_devnet_client_from_predefined_account(self) -> AccountClient:
+        key_pair = KeyPair.from_private_key(int(ACCOUNT_CLIENT_PRIVATE_KEY, 0))
```

Comment:
Let's replace `make_devnet_client` with that one (contents only) - and then there's no need for this other method